config/crd/bases/cluster.x-k8s.io_machines.yaml

```diff
@@ -32,7 +32,6 @@ spec:
     shortNames:
     - ma
     singular: machine
-  preserveUnknownFields: false
```

Comment:
We likely don't want to drop this, it may require updating the kubebuilder annotation for Machine to set `crd:preserveUnknownFields=false` as per https://github.com/kubernetes-sigs/controller-tools/pull/362